libraries/commerce/product/selectors/relations.js

```diff
@@ -0,0 +1,60 @@
+import { createSelector } from 'reselect';
+import { generateProductRelationsHash } from '../helpers';
+import { getProducts } from './product';
+
+/**
+ * Returns productRelations state.
+ * @param {Object} state State.
+ * @returns {Object}
+ */
+export const getProductRelationsState = state => state.product.productRelations;
+
+/**
+ * Returns product relations state for given hash.
+ * @param {string} hash State hash.
+ * @returns {function|undefined}
+ */
+export const getProductRelationsByHash = hash => createSelector(
```

Comment:
Yes I did consider documentation. I'll make this selector private since it's not intended to be used outside. Developer should only use getProductRelations and getRelatedProducts in the end. Hashing is suppose to be internal layer.